src/IFX007T-Motor-Control.cpp

```diff
@@ -165,27 +231,38 @@ void IFX007TMotorControl::configureBLDCMotor(BLDCParameter MyParameters)
     _debugPin = 1;
     
 
-    if(MyParameters.SensingMode)    //Hall-sensor mode (Not supported yet)
+    if(MyParameters.SensingMode)    //Hall-sensor mode
     {
         #define HALLmode
```

Comment:
Defines immer alles GROSS schreiben z.B. #define KILIAN "Ist die Nummer 1"

Reply:
Good to know, I changed the defines for the pin assignment as well to capital letters 👍